src/MeasureTheory.jl

```diff
@@ -119,6 +118,9 @@ xlog1py(x, y) = x * log(1 + y)
 
 as(args...; kwargs...) = TV.as(args...; kwargs...)
```

Comment:
so this `as` shadows the original `TV.as`, i.e. overloads of `as` will not become overloads of `TV.as`, is that intended?

Reply:
Good catch, yes this is a little weird. I shadowed it to avoid type piracy, in particular in cases where I defined `as` taking a `Distribution` argument. I think this will all soon go away, because we'll only need Distributions.jl for some sampling and logpdf code, and we'll move TransformVariables to an extension.